ztp/gitops-subscriptions/argocd/PowerConfiguration.md

```diff
@@ -0,0 +1,190 @@
+      policyName: "config-policy"
+      spec:
+        profile:
+          - name: performance-patch
+            data: |
+              ...
+              [sysfs]
+              /sys/devices/system/cpu/intel_pstate/max_perf_pct=72
+```
+
+More information pertaining to the Power saving configuration can be found in the official [OpenShift docs](https://docs.openshift.com/container-platform/4.12/scalability_and_performance/cnf-low-latency-tuning.html#node-tuning-operator-pod-power-saving-config_cnf-master).
+
+####  [Optional: Power saving mode](#optional-power-saving-mode)
+
+For critical workloads that require the highest performance and lowest latency, each such pod can be annotated to disable C-states and the cpufreq governor set to `performance` as shown below:
+
+```yaml
+metadata:
+  annotations:
+    cpu-c-states.crio.io: "disable"
+    cpu-freq-governor.crio.io: "performance"
+```
+
+It is worthwhile to note the following:
+
+- These annotations are likely to be used with the other high performance cri-o annotation described [here](https://docs.openshift.com/container-platform/4.12/scalability_and_performance/cnf-low-latency-tuning.html#node-tuning-operator-pod-power-saving-config_cnf-master).
+- These annotations have the same restrictions as the other high performance cri-o annotations:
+    - The pod must use the performance-&lt;profile-name&gt; runtime class.
+    - The pod must have a [QoS Class of guaranteed](https://kubernetes.io/docs/tasks/configure-pod-container/quality-service-pod/#create-a-pod-that-gets-assigned-a-qos-class-of-guaranteed) (i.e. uses whole CPUs).
+
+## Migrating from one mode to another
+
+This section outlines the steps and configurations for migrating from one mode to another mode.
+
+### 1) Performance to Power saving
+
+Migrating from a Performance configuration to a Power saving configuration requires setting the workloadHints parameters as follows:
+
+```yaml
+workloadHints:
+  realTime: true
+  highPowerConsumption: false
+  perPodPowerManagement: true
+```
+
+> **Note**: Critical workload pods should be annotated with the [crio annotations](#optional-power-saving-mode) shown above prior to the migration.
+
+It is recommended to set a cpu governor in the additional kernel arguments specification as well as capping the max cpu frequency as explained in the [power saving mode](#3-power-saving-mode).
+
+After the above policy changes have been made, it should be pushed to the Git repository and synced through argocd policy application. If the policy has a `remediationAction` set to `enforce`, then TALM (topology aware lifecycle manager) should do the needful of applying the policy to the effected SNOs. If however the `remediationAction` is configured for `inform`, then a CGU (cluster group upgrade) CR must be created and applied to the hub cluster.
+
+After the policy has been applied to the effected SNOs, the change can be verified by viewing the PerformanceProfile on the SNOs:
```

Comment:
This seems like it goes too deeply into the "how the gitops/policy solution manages configuration". Maybe keep it focused on the technical changes here and defer to other docs for how it works.
```suggestion
When using the recommended GitOps and Policy solution for managing cluster configuration, the above changes need to be propagated to the cluster(s) to take effect. Note that any changes to the PerformanceProfile and/or kernel arguments will results in a reboot of the affected nodes.

After the changes have been applied to the affected SNOs, the change can be verified by viewing the PerformanceProfile on the SNOs:
```